server/src/services/proposal.services.js

```diff
@@ -58,8 +58,12 @@ export const postNewProposal = (title, type, description, level, expiration_date
             reject(err);
           }
           const propId = row.id;
-          sqlSuper.run(propId, supervisor_obj.supervisor_id, supervisor_obj.co_supervisor_id || null, supervisor_obj.external_supervisor_id || null)
-          sqlSuper.finalize();
+          if (supervisor_obj.co_supervisors && supervisor_obj.co_supervisors.length>0) {
```

Comment:
In this file the green part can be accepted because I think that are the updates made by you or Oscar